# ==============================================================================
# REGRESSOR COMMIT DIFF
# ==============================================================================
# File: third_party/rust/rose_tree/tests/walkers.rs
# Commit: ef5dc3e04e5f
# Full Hash: ef5dc3e04e5f271eea0636ab3a495e95cc912f1d
# Author: Dzmitry Malyshau <dmalyshau@mozilla.com>
# Date: 2021-09-04 09:40:24
# Regressor Bug: 1726626
# File Overlap Count: 3
# Description:
#   Bug 1726626 - Move gfx/wgpu into a 3rd party dependency r=jgilbert,bholley
#   
#   This update makes wgpu a vendored dependency instead of having it in gfx/wgpu.
#   
#   ## Notes
# ==============================================================================

diff --git a/third_party/rust/rose_tree/tests/walkers.rs b/third_party/rust/rose_tree/tests/walkers.rs
deleted file mode 100644
--- a/third_party/rust/rose_tree/tests/walkers.rs
+++ /dev/null
@@ -1,50 +0,0 @@
-
-extern crate rose_tree;
-
-use rose_tree::RoseTree;
-
-struct Weight;
-
-
-#[test]
-fn walk_children() {
-
-    let (mut tree, root) = RoseTree::<Weight, u32>::new(Weight);
-    let a = tree.add_child(root, Weight);
-    let b = tree.add_child(root, Weight);
-    let c = tree.add_child(root, Weight);
-
-    let mut child_walker = tree.walk_children(root);
-    assert_eq!(Some(c), child_walker.next(&tree));
-    assert_eq!(Some(b), child_walker.next(&tree));
-    assert_eq!(Some(a), child_walker.next(&tree));
-    assert_eq!(None, child_walker.next(&tree));
-
-    let d = tree.add_child(b, Weight);
-    let e = tree.add_child(b, Weight);
-    let f = tree.add_child(b, Weight);
-
-    child_walker = tree.walk_children(b);
-    assert_eq!(Some(f), child_walker.next(&tree));
-    assert_eq!(Some(e), child_walker.next(&tree));
-    assert_eq!(Some(d), child_walker.next(&tree));
-    assert_eq!(None, child_walker.next(&tree));
-}
-
-
-#[test]
-fn walk_siblings() {
-
-    let (mut tree, root) = RoseTree::<Weight, u32>::new(Weight);
-    let a = tree.add_child(root, Weight);
-    let b = tree.add_child(root, Weight);
-    let c = tree.add_child(root, Weight);
-    let d = tree.add_child(root, Weight);
-
-    let mut sibling_walker = tree.walk_siblings(a);
-    assert_eq!(Some(d), sibling_walker.next(&tree));
-    assert_eq!(Some(c), sibling_walker.next(&tree));
-    assert_eq!(Some(b), sibling_walker.next(&tree));
-    assert_eq!(None, sibling_walker.next(&tree));
-}
-